app/models/index.js

```diff
@@ -20,10 +20,10 @@ db.sequelize = sequelize;
 
 db.user = require("./user.model.js")(sequelize, Sequelize);
 db.story = require("./stories.model.js")(sequelize, Sequelize);
-db.characterRole = require("./characterRole.model.js")(sequelize, Sequelize);
-db.country = require("./country.model.js")(sequelize, Sequelize);
-db.language = require("./language.model.js")(sequelize, Sequelize);
-db.setting = require("./setting.model.js")(sequelize, Sequelize);
+db.characters = require("./characters.model.js")(sequelize, Sequelize);
+db.countries = require("./countries.model.js")(sequelize, Sequelize);
+db.languages = require("./languages.model.js")(sequelize, Sequelize);
+db.settings = require("./setting.model.js")(sequelize, Sequelize);
```

Comment:
probably worth thinking about renaming files to the same plural form, e.g., why is it `settings.controller.js` but only `setting.model.js`, but then back to plural `settings.routes.js` again?

Reply:
Changed it! this must've slipped by me. We are trying to maintain a plural consistency.